dvc/output/base.py

```diff
@@ -86,13 +97,26 @@ def __init__(
                 )
             )
 
+        self.path_info = self._parse_path(remote, path)
+
+    def _parse_path(self, remote, path):
+        if remote:
```

Comment:
does remote always set?

Reply:
No, `remote` is only set when it's `remote://name/...`. `self.remote` is, however, always set, even if that is a "hanging in the air remote", with `path_info == None`.